src/helper.cc

```diff
@@ -1207,8 +1207,10 @@ Enqueue(helper * hlp, Helper::Xaction * r)
 
     /* do this first so idle=N has a chance to grow the child pool before it hits critical. */
     if (hlp->childs.needNew() > 0) {
-        debugs(84, DBG_CRITICAL, "Starting new " << hlp->id_name << " helpers...");
-        helperOpenServers(hlp);
+        CallService(nullptr, [&hlp] {
+            debugs(84, DBG_CRITICAL, "Starting new " << hlp->id_name << " helpers...");
+            helperOpenServers(hlp);
+        });
```

Comment:
Please check whether StatefulEnqueue() should do the same (when calling helperStatefulOpenServers() at least). The two functions seem to be rather symmetric in purpose/nature...